packages/gatsby-source-mongodb/src/utility.js

```diff
@@ -0,0 +1,5 @@
+function sanitizeName(s) {
```

Comment:
Maybe just rename this file sanitize-name? And use typical `module.exports` instead of `exports.sanitizeName`?

I like abstracting this into a separate file to make it easier to test, but I think we can improve this a bit!